src/highlevel.jl

```diff
@@ -759,15 +759,21 @@ function activate!(s::Stream; flags = 0, timens = nothing, numElems = nothing)
     return nothing
 end
 
-function activate!(f::Function, s::Stream; kwargs...)
+function activate!(f::Function, streams::AbstractVector{<:Stream}; kwargs...)
+    activated_streams = SoapySDR.Stream[]
```

Comment:
I dislike that this allocates, and is now the default code path. I think people are going to have a hard enough time handling JIT latency after stream activation in their SDR tasks, and this would further compound the problem (tough minor, but still not in the correct direction). Anyway, this seems to be a convenience for a minority with multiple SDRs. I think it is likely the majority of users use only a single SDR and it would be better to figure out how to get SoapyMultiSDR_jll working.

Reply:
I got SoapyMultiSDR_jll working (I could open the device) but then it failed with a mysterious internal error about MMAP when I tried to use it.  So we may be able to get it to work, but it may take some debugging.

With regards to allocations; this is one allocation per session; that's a very small cost, IMO.